### PR TITLE
bpo-5430: Preservation LF, CR in IMAP4 append method.

### PR DESCRIPTION
**Pull request by hobbestigrou**
Preserve LF, CR in the append method.

Co-authored-by: Ron DuPlain <ron.duplain@gmail.com>

https://bugs.python.org/issue5430

### CODE REVIEW

File: Lib/imaplib.py
Comment (JulienPalard) [inline Lib/imaplib.py:114]: In any cases, please don't remove it, [one](https://github.com/gaubert/gmvault/blob/61a3f633a352503a395b7f5a0d9cf5fc9e0ac9b4/src/gmv/mod_imap.py#L288) may be using it, please do as the previous `Literal` one, keep it but state it's no longer used and kept for backward compatibility.

**Review by JulienPalard — CHANGES_REQUESTED** [review]
Hi Natal! This PR is in complete contradiction with: https://bugs.python.org/issue723962

So I think it needs more discussion on the issue before being merged, and will probably needs to be modified before being merged (though the "Request changes").

**Comment by AlexWaygood** [comment]
Changes were requested over two years ago by a core dev, but have not been applied. I am therefore closing this PR.

@hobbestigrou, if you're still interested in working on this issue, feel free to open a new PR. Alternatively, ping me and I'll happily reopen the PR. Thanks! 🙂

**Pull request closed without merging**
